index.js

```diff
@@ -57,25 +57,36 @@ module.exports = function implicitFiguresPlugin(md, options) {
       // for linked images, image is one off
       image = token.children.length === 1 ? token.children[0] : token.children[1];
 
-      if (options.figcaption == true) {
-        if (image.children && image.children.length) {
+      if (options.figcaption) {
+        let figCaption;
+        const captionObj = image.attrs.find(([k]) => k === 'title');
```

Comment:
I prefer `var` and `function` over `let`, `const` and `() => {}`, to be backward compatible.

Reply:
Hi @arve0 ! I did that to mimic what was being done on the `copyAttrs` area (now around line 85). Do you want me to change it everywhere? I'm not sure about backward compatibility. Right now that wouldn't work on 0.10.0 either since it's using array destructuring as well. Do you want me to go and fix those? More than happy to do so!